src/frontend/js/modules/search/sub-header/sub-header.pug

```diff
@@ -3,4 +3,4 @@
     .flex-wrap.flex-vertical-centered.all-centered.module-subheader-inner.hidden-xs
       span.title.ellipsis.all-centered {{ 'Global Search' | translate }}
     .module-subheader-inner.visible-xs
-      esn-search-header
+      esn-search-header(ng-if='showGlobalSearch')
```

Comment:
Please check this case with  `contextual sidebar` on mobile

Reply:
Thanks. I updated it and retested with mobile page